Couldn't Import IonSFUGRPCWebSignal

<!--
  Come join our community on Slack https://pion.ly/slack for quicker support and more resources

  We also provide a Knowledge Base https://pion.ly/knowledge-base that covers Pion and general WebRTC
  
  We maintain a FAQ of common issues and questions https://pion.ly/knowledge-base/pion-basics/faq/

 Please answer these questions before submitting your issue. Thanks!
-->

Hello, I'm testing out this SDK on a React application, with the intent of connecting to my SFU using gRPC but am having issues. 

### What did you do?
<!--
If possible, provide an example for reproducing the error.
A complete runnable program is good.
-->

  1) Ran `npm install ion-sdk-js` 
  2) Ran the following code
![image](https://user-images.githubusercontent.com/37765758/106066256-5ebf3800-60b1-11eb-96d9-32435c93c2f7.png)
  3) Got an error `Module not found: Can't resolve './_proto/library/sfu_pb' in '/Users/dantruong/projects/video-test/node_modules/ion-sdk-js/lib/signal'`

### What did you expect?
I expected the import to work. 

### What happened?
I checked the node_modules and saw that the following was the ion-sdk-js directory structure: 
![image](https://user-images.githubusercontent.com/37765758/106066345-83b3ab00-60b1-11eb-9eb8-1fb715f7c1b1.png)

For some reason, there is no `ion-sdk-js/lib/signal/_proto/library/sfu_pb` even though the program expects it. So my guess is that npm didn't install the library properly?

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "lib/**/*"
   ],
   "scripts": {
-    "addProtosToBuild": "mkdir ./lib/signal/_proto && mkdir ./lib/signal/_proto/library && cp -r ./src/signal/_proto/library ./lib/signal/_proto",
+    "addProtosToBuild": "[ -d ./lib/signal/_proto/library ] || (mkdir ./lib/signal/_proto && mkdir ./lib/signal/_proto/library) && cp -r ./src/signal/_proto/library ./lib/signal/_proto",
     "build": "tsc && npm run addProtosToBuild",
     "bundle": "webpack",
     "dev": "tsc -w",

--- a/package.json
+++ b/package.json
@@ -9,7 +9,8 @@
     "lib/**/*"
   ],
   "scripts": {
-    "build": "tsc",
+    "addProtosToBuild": "mkdir ./lib/signal/_proto && mkdir ./lib/signal/_proto/library && cp -r ./src/signal/_proto/library ./lib/signal/_proto",
+    "build": "tsc && npm run addProtosToBuild",
     "bundle": "webpack",
     "dev": "tsc -w",
     "format": "prettier --write \"src/**/*.ts\"",

